PowerNSX.psm1

```diff
@@ -20212,6 +20212,76 @@ function Remove-NsxIpSet {
 }
 
 
+function Remove-NsxIpPool {
+
+    <#
+    .SYNOPSIS
+    Removes the specified NSX IPPool.
+
+    .DESCRIPTION
+    An IP Pool is a simple IPAM construct in NSX that simplifies automated IP
+    address asignment for multiple NSX technologies including VTEP interfaces
+    NSX Controllers.
+
+    This cmdlet removes the specified IP Pool. If the object 
+    is currently in use the api will return an error.
+
+    .EXAMPLE
+    PS C:\> Get-NsxIPPool TestIPPool | Remove-NsxIPPool
+
+    #>
+ 
+    param (
+
+        [Parameter (Mandatory=$true,ValueFromPipeline=$true,Position=1)]
+            [ValidateNotNullOrEmpty()]
+            [System.Xml.XmlElement]$IPPool,
+        [Parameter (Mandatory=$False)]
+            #Prompt for confirmation.  Specify as -confirm:$false to disable confirmation prompt
+            [switch]$Confirm=$true,
+        [Parameter (Mandatory=$False)]
+            #PowerNSX Connection object.
+            [ValidateNotNullOrEmpty()]
+            [PSCustomObject]$Connection=$defaultNSXConnection
+
```

Comment:
No problem - it wasn't that long ago that I was was trying to comprehend the same thing! Just update the branch in your fork (rhancock8:master) with new commits and they should automatically get added to this PR.